flask_resty/testing.py

```diff
@@ -61,15 +61,30 @@ def assert_shape(actual, expected):
         # superset of the expected items, rather than that they match.
         for key, value in expected.items():
             if value is not UNDEFINED:
-                assert key in actual
+                assert (
+                    key in actual
+                ), f"Expected key {key} not found in: {actual}"
```

Comment:
nitpick: keep casing consistent. also probably makes sense to use `reprs`.

```suggestion
                ), f"expected key {key!r} not found in: {actual!r}"
```